app/serializers/shop_advertisement_serializer_mixin.rb

```diff
@@ -8,15 +8,19 @@ def self.included(klass)
   def shop_advertisement_html
     current_user = object&.guardian&.user&.single_sign_on_record&.external_id
 
-    best_answer_id = object.topic.custom_fields["accepted_answer_post_id"]
+    best_answer_id = object&.topic&.custom_fields["accepted_answer_post_id"]
 
     target_user_id = best_answer_id ? Post.find(best_answer_id).user_id : user_id
 
-    shop_id = UserCustomField.where(user_id: target_user_id, name: 'storeId').first&.value ||
-              Discourse.cache.read(:store_ids_for_advertisement)&.sample
+    if object&.topic&.user&.groups&.exists?(:name => SiteSetting.DiscourseDesignBundles_trusted_group)
+      return ''
+    else
+      shop_id = UserCustomField.where(user_id: target_user_id, name: 'storeId').first&.value ||
+      Discourse.cache.read(:store_ids_for_advertisement)&.sample
```

Comment:
Fix padding, so it's clear that it's a new line of expression rather then just a separate line